fair/run.py

```diff
@@ -195,6 +208,7 @@ def run_command(
     _email = _glob_conf['user']['email']
 
     if mode == CMD_MODE.RUN:
+        _logger.debug("Performing 'register' substitutions")
         _conf_yaml = fdp_reg.subst_registrations(local_uri, _work_cfg)
     else:
```

Comment:
Won't this mean that the `--ci` mode won't create the working config, or am I misunderstanding this? We still want everything to work except just the execution of the script, so that R's CI can call `fair pull in.yaml` and `fair run --ci in.yaml` from inside R, and then just continue with the script to avoid having to restart R lots of times to run the testing. So the working config and the script still needs to be created.

Reply:
This is a good point and does need a change, the reason for the `if` originally was so it would not be called if `PUSH` or `PULL` are the mode. As we now have a fourth `PASS` then that also needs to be in this `if` statement yes...